package.json

```diff
@@ -28,15 +27,15 @@
     "ganache": "ganache-cli --networkId ${npm_package_config_ganacheNetworkID} --port ${npm_package_config_ganachePort} --gasLimit ${npm_package_config_ganacheGasLimit} --gasPrice ${npm_package_config_ganacheGasPrice} --defaultBalanceEther ${npm_package_config_etherBalance} --deterministic --mnemonic \"${npm_package_config_mnemonic}\" ${npm_package_config_extra}",
     "ganache:verbose": "ganache-cli --networkId ${npm_package_config_ganacheNetworkID} --verbose --port ${npm_package_config_ganachePort} --gasLimit ${npm_package_config_ganacheGasLimit} --gasPrice ${npm_package_config_ganacheGasPrice} --defaultBalanceEther ${npm_package_config_etherBalance} --deterministic --mnemonic \"${npm_package_config_mnemonic}\" ${npm_package_config_extra}",
     "ganache:stop": "ps aux | grep ganache-cli | grep -v grep | awk '{print $2}' | xargs kill -9",
-    "deploy:ganache": "buidler run --network ganache src/scripts/deploy-contracts.ts",
-    "deploy": "buidler run src/scripts/deploy-contracts.ts --network"
+    "deploy:ganache": "hardhat run --network ganache src/scripts/deploy-contracts.ts",
+    "deploy": "hardhat run src/scripts/deploy-contracts.ts --network"
   },
   "files": [
     "artifacts",
     "typings"
   ],
   "config": {
-    "mnemonic": "concert load couple harbor equip island argue ramp clarify fence smart topic",
+    "mnemonic": "test test test test test test test test test test test junk",
```

Comment:
why? shouldn't we either remove this "config" entry, and use hardhat.config.ts mnemonic, or put something that works?

Reply:
this is for ganache

Reply:
Does that “test test .. “ work on ganache?

Reply:
yea its valid mnemonic, its nice because it's self explanatory that it is a test config

Reply:
Neat